src/backend/base/langflow/api/v1/chat.py

```diff
@@ -179,7 +179,7 @@ async def build_vertex(
             result_data_response = ResultDataResponse(**result_dict.model_dump())
 
         except Exception as exc:
-            logger.exception(f"Error building vertex: {exc}")
+            logger.exception(f"Error building langflow vertex: {exc}")
```

Comment:
I'd say we just make it `Error building Component: {exc}`

Reply:
done